Common/KeyMap.cpp

```diff
@@ -980,6 +985,7 @@ void AutoConfForPad(const std::string &name) {
 	// Add a couple of convenient keyboard mappings by default, too.
 	g_controllerMap[VIRTKEY_PAUSE].push_back(KeyDef(DEVICE_ID_KEYBOARD, NKCODE_ESCAPE));
 	g_controllerMap[VIRTKEY_UNTHROTTLE].push_back(KeyDef(DEVICE_ID_KEYBOARD, NKCODE_TAB));
+	g_controllerMap[VIRTKEY_FRAME_ADVANCE].push_back(KeyDef(DEVICE_ID_KEYBOARD, NKCODE_F));
```

Comment:
Ah, right, I thought it kept the flag set that it was advancing frame by frame for some reason.  But good to remove the default still, indeed.

-[Unknown]